.github/workflows/osv-scanner.yaml

```diff
@@ -0,0 +1,40 @@
+name: OSV-Scanner
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: 21 8 * * 1
+  workflow_dispatch: {}
+
+permissions:
+  actions: read
+  contents: read
+  security-events: write
+
+jobs:
+  scan-scheduled:
+    name: OSV scan scheduled
+
+    if: github.event_name == 'push' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    uses: google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@764c91816374ff2d8fc2095dab36eecd42d61638 # v1.9.2
```

Comment:
what are you trying to do with these actions? This is going to store just the manifest, it's not code...

Reply:
Please, excuse me. It was a part of our template for all repositories under github.com/freelensapp and of course it is unnecessary for this package.

Already removed.